src/test/java/seedu/taassist/storage/JsonAdaptedStudentTest.java

```diff
@@ -100,8 +100,8 @@ public void toModelType_nullAddress_throwsIllegalValueException() {
 
     @Test
     public void toModelType_invalidTags_throwsIllegalValueException() {
```

Comment:
may be can refactor all `Tags` to `Classes`?

Reply:
This is mentioned in another issue regarding refactoring tags to classes.